cli/scripts/monorepo.ts

```diff
@@ -356,7 +356,7 @@ fs.copyFileSync(
           ...options,
         });
       case "npm":
-        return execa.sync("npm", ["run", command, ...(args || [])], {
+        return execa.sync("npm", [command, ...(args || [])], {
```

Comment:
Curious about this one, don't we still need `run` for `npm`?

Reply:
I haven't dug through the git history, but I assume that this function was originally used to run package scripts in tests. Currently in our e2e tests we only use it to run install commands and script execution uses `Monorepo.turbo`